db/seeds.rb

```diff
@@ -26,24 +25,24 @@
 
 36.times do
   Costume.create!(
-    name: Faker::Name.name,
+    name: Faker::Lorem.paragraph,
     description: "lorem ipsum",
```

Comment:
sorry! i made the comment on the wrong line....
line 29: name: Faker::Name.name
line 30: description: Faker::Lorem.paragraph

Reply:
On it!